Library/Homebrew/test/test_cmd_audit.rb

```diff
@@ -301,10 +301,10 @@ class Foo < Formula
   end
 
   # Regression test for https://github.com/Homebrew/homebrew/pull/48744
-  # Formulae with "++" in their name would break the name check because of a
-  # regexp error:
+  # and https://github.com/Homebrew/homebrew/pull/48744
```

Comment:
This line can be omitted; it's the same URL that is already referenced in the line above this one. :wink: